tests/llvm/test_builtins_vector.py

```diff
@@ -42,3 +45,29 @@ def test_vector_op(benchmark, op, y, llvm_y, builtin, result, mode):
         benchmark(llvm_fun, ct_u, llvm_y, DIM_X, ct_res)
         res = llvm_res
     assert np.allclose(res, result)
+
+
+@pytest.mark.benchmark(group="VecOuter")
+def test_vec_outer_numpy(benchmark):
+    numpy_res = benchmark(np.outer, u, w)
+    assert np.allclose(numpy_res, outer_res)
+
+
+@pytest.mark.llvm
+@pytest.mark.benchmark(group="VecOuter")
+def test_vec_outer_llvm(benchmark):
+    llvm_fun = pnlvm.LLVMBinaryFunction.get("__pnl_builtin_vec_outer_product")
+    benchmark(llvm_fun, ct_u, ct_w, DIM_X, DIM_Y, ct_outer_res)
+    assert np.allclose(llvm_outer_res, outer_res)
+
+
+@pytest.mark.llvm
+@pytest.mark.cuda
+@pytest.mark.benchmark(group="VecOuter")
+def test_vec_outer_cuda(benchmark):
```

Comment:
Has this been tested on a CDUA machine, or does it need a test run?

Reply:
It has not, since I don't have access to a machine with CUDA set up

Reply:
OK, I'll test it on one of mine